test/e2e.test.js

```diff
@@ -0,0 +1,862 @@
+/* eslint-disable import/no-commonjs */
+require('@babel/polyfill');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const webpack = require('webpack');
+const puppeteer = require('puppeteer');
+const { PendingXHR } = require('pending-xhr-puppeteer');
+
+const utils = {};
+
+utils.getParamsFromRequest = request => {
+  const { _postData } = request;
+  const { params } = JSON.parse(_postData);
+  return new URLSearchParams(params);
+};
+
+utils.getXHRDataForQuery = (pendingXHR, query, source = 'places') => {
+  const finishedXHRs = pendingXHR.finishedWithSuccessXhrs;
+  const extractQueryFromRequest = request =>
+    utils.getParamsFromRequest(request).get('query');
+
+  return Array.from(finishedXHRs).find(request => {
+    return (
+      extractQueryFromRequest(request) === query && request._url.match(source)
+    );
+  });
+};
+
+utils.getResponseBodyFromRequest = request => {
+  return request._response.json();
+};
+
+utils.matchDomWithHit = (domText, hit) => {
+  const textContent = domText.trim();
+
+  return [
+    textContent.match(hit.locale_names[0]),
+    textContent.match(hit.city[0]),
+    textContent.match(hit.country),
+  ].every(Boolean);
+};
+
+describe('releases', () => {
+  beforeAll(() => {
+    // check that the build dist exists
+    const distExists = fs.existsSync(path.join(__dirname, '..', 'dist'));
+    const indexExists = fs.existsSync(
+      path.join(__dirname, '..', 'dist', 'index.js')
+    );
+    const cdnExists = fs.existsSync(path.join(__dirname, '..', 'dist', 'cdn'));
+
+    assert.equal(distExists, true, 'Expected dist folder to exist');
+    assert.equal(indexExists, true, 'Expected npm entrypoint to exist');
+    assert.equal(cdnExists, true, 'Expected cdn folder to exist');
+  });
+
+  describe('npm package', () => {
+    let url;
+    let browser;
+    let page;
+
+    beforeAll(done => {
+      url = `file://${path.join(__dirname, 'npm-lib', 'index.html')}`;
+      // webpack bundle
+      webpack(
```

Comment:
do you test only via webpack, or also just a script tag?

Reply:
The npm lib is tested via webpack (because it has to be included in some other script to behave properly)
The cdn examples are all tested via script tag.